src/utils.jl

```diff
@@ -5,3 +5,25 @@ function masked_crc32c(data)
     x = UInt32(CRC32c.crc32c(data))
     return UInt32(((x >> 15) | UInt32(x << 17)) + 0xa282ead8)
 end
+
+function serialize_proto(data::Union{ProtoType, ProtoEnum})
+    pb = PipeBuffer()
+    writeproto(pb, data)
+    pb.data
+end
+
+"""
+    markdown_repr(x)
+
+Returns the most suitable method for displaying `x` in a markdown document.
+"""
```

Comment:
See improved version in:
https://discourse.julialang.org/t/how-to-get-a-markdown-repr-of-an-object-mime-type-fallbacks/23242/3?u=oxinabox